### DISCUSSION TITLE
Request for issue Hosting Review

### DISCUSSION THREAD

**Discussion by aprasanth3192** [community/community#140556] [New to GitHub]
Hello, I was creating a new issue in our Jenkins CI repository from our GitHub organization at 2 weeks before. 
But in this creating issue ticket isn't closed now. How much time the Hosting team to taken for review in this creating new issue in us 
Jenkins CI repository from our GitHub organization.

**Comment by antidude900** [comment]
The time it takes for an issue to be reviewed depends on the workload of the hosting team and the nature of the issue. If the issue is really important, then they will review it asap. But if they don't find it important, they will simply ignore it.

In the meantime, you can try tagging the appropriate team members or maintainers in the issue to bring it to their attention. Also, make sure the issue includes all relevant details like logs, screenshots, and steps to reproduce to make the review smoother.
